tox.ini

```diff
@@ -2,6 +2,7 @@
 skipsdist = True
 
 [testenv]
+basepython = python3
```

Comment:
Are we actually running into an issue with this? I haven't had to use this in any other `tox.ini` file that I've written/used.

Reply:
I inherited this from [alertmanager](https://github.com/canonical/alertmanager-operator/blob/main/tox.ini#L16) and didn't pay much attention to it. We can remove it.

Reply:
Removed. Closing this.